docs/app/pages/components/components-sections/hierarchy-bar/hierarchy-bar/snippets/icon.html

```diff
@@ -0,0 +1,3 @@
+<ux-hierarchy-bar [root]="node" [mode]="mode" [(selected)]="selected">
+    <i class="hpe-icon hpe-user-manager" *uxHierarchyBarNodeIcon="let node"></i>
```

Comment:
Could we have an example that makes use of the `node` parameter here?

Reply:
Sure, updated it to add a class based on the `node.icon` property